src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResource.kt

```diff
@@ -73,12 +73,12 @@ class UserResource(
     @PathVariable @Size(max = 30, min = 1, message = "username must be between 1 and 30") username: String
   ) = userService.deleteUser(username)
 
-  @PreAuthorize("hasRole('ROLE_MAINTAIN_ACCESS_ROLES_ADMIN') or hasRole('ROLE_MAINTAIN_ACCESS_ROLES')")
+  @PreAuthorize("hasRole('ROLE_MANAGE_NOMIS_USER_ACCOUNT') or hasRole('ROLE_MAINTAIN_ACCESS_ROLES')")
   @GetMapping("/{username}")
```

Comment:
I think it requires all 3:

- ROLE_MAINTAIN_ACCESS_ROLES_ADMIN
- ROLE_MAINTAIN_ACCESS_ROLES
- ROLE_MANAGE_NOMIS_USER_ACCOUNT